linchpin/__init__.py

```diff
@@ -109,6 +109,7 @@ def __init__(self, ctx):
         self.set_evar('from_api', True)
         self.workspace = self.get_evar('workspace')
         self.disable_pbar = 'True'
+        self.ctx.no_monitor = False
```

Comment:
You'll need to check if ctx has already set monitor before setting it to false

Reply:
Also, you'll have to set the evar `no_monitor` because it needs to be available in the ZMQ CallbackPlugin